staging/src/k8s.io/apimachinery/pkg/api/meta/conditions.go

```diff
@@ -27,17 +27,17 @@ import (
 //  1. if the condition of the specified type already exists (all fields of the existing condition are updated to
 //     newCondition, LastTransitionTime is set to now if the new status differs from the old status)
 //  2. if a condition of the specified type does not exist (LastTransitionTime is set to now() if unset, and newCondition is appended)
-func SetStatusCondition(conditions *[]metav1.Condition, newCondition metav1.Condition) {
+func SetStatusCondition(conditions *[]metav1.Condition, newCondition metav1.Condition) (changed bool) {
```

Comment:
Update godoc to something like:

`// SetStatusCondition sets the corresponding condition in conditions to newCondition and returns true if the conditions are changed by this call.` ?

Reply:
Done